doc/md/fundamentals/2-types/12-advanced-types.md

```diff
@@ -10,13 +10,13 @@ Advanced type features enable more flexible and expressive type definitions, inc
 
 Structural equality determines whether two values are equal based on their contents. This applies to immutable data structures, such as [records](https://internetcomputer.org/docs/motoko/fundamentals/types/records) and [variants](https://internetcomputer.org/docs/motoko/fundamentals/types/variants), but does not apply to mutable structures for safety reasons.
 
-```motoko no-repl
+```motoko
 type Point = { x : Int; y : Int };
 
 let p1 : Point = { x = 1; y = 2 };
 let p2 : Point = { x = 1; y = 2 };
 
-let areEqual = p1 == p2;  // true (structural equality)
+p1 == p2;  // true (structural equality)
 ```
 
 Even though `p1` and `p2` are distinct objects, they are considered equal because they have the same structure and values.
```

Comment:
This remains true even if we add some different fields to the point values, since the `==` on Point  values only considers the `x` and `y` fields.

```motoko
type Point = { x : Int; y : Int };
let p1 : Point = { x = 1; y = 2; z = 3 };
let p2 : Point = { x = 1; y = 2; z = 4; c = "Red"; };
p1 == p2;  // true (structural equality at type `Point`)
```